src/util/fetchPokemonData.js

```diff
@@ -0,0 +1,12 @@
+const baseUrl = `https://pokeapi.co/api/v2/pokemon/`;
+
+export const fetchPokemonDataByName = async (name) => {
```

Comment:
Good job. I think we should rename the parameter to `searchInput` or something, since user can search by `id` too, and not just by Pokemon name. 

Same for the `nameUrl`. Perhaps replace it with `pokemonSearchUrl` or something.

Reply:
will do!